src/workflow.scss

```
@@ -162,6 +166,10 @@
   color: #76BB21 !important;
 }
 
+.workflow-extension-google-popup-footer-ebsco a {
+  color: #002F56 !important;
+}
+
```

is it's the same property it could be together ?
```suggestion
.workflow-extension-google-popup-ebsco .workflow-extension-google-popup-subheader a,
.workflow-extension-google-popup-footer-ebsco a { 
 color: #002F56 !important;
 }
```